### PR TITLE
fix: User billing details for paying event invoice

### LINKED ISSUE #6039: Adding fields for organizer billing info
**Describe the feature**
Related Frontend Issue:https://github.com/fossasia/open-event-frontend/issues/3117
There is a requirement to submit the billing info so that the organiser can pay the invoice. For this, additional fields need to be added in the user model.

(Add fields for the following information: Contact Name*, Phone, Email (Use account email here), Country*, Tax ID, Registered Entity or Company, Business Address*, City*, ZIP Code*, Additional Info)

**Additional context**
Working on this

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #6039 
**REQUIRES MERGE OF #6098**

Short description of what this resolves:

- Add model/schema additions for payment details
- Migration added
- Followed billing name convention

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/schema/users.py [hunk 1]
```diff
@@ -69,6 +69,17 @@ class Meta:
     deleted_at = fields.DateTime(dump_only=True)
     details = fields.Str(allow_none=True)
     contact = fields.Str(allow_none=True)
+    contact_name = fields.Str(allow_none=True)
+    phone = fields.Str(allow_none=True)
+    email = fields.Str(allow_none=True)
```
Comment (CosmicCoder96) [inline app/api/schema/users.py]: what is this? Users already have an email? You are adding a field called email in to the user schema! Users already have an email, that's how we identify them?

Comment (mrsaicharan1) [reply app/api/schema/users.py]: @CosmicCoder96 Yes, I agree. But I think that sometimes users might prefer to give another email address for their billing information?
But either ways, the names would result in a conflict. Got that. I'll rename them as something like`billing_email`

Comment (CosmicCoder96) [reply app/api/schema/users.py]: No, the issue says this:
(Add fields for the following information: Contact Name*, Phone, Email (**Use account email here**), Country*, Tax ID, Registered Entity or Company, Business Address*, City*, ZIP Code*, Additional Info)

Comment (mrsaicharan1) [reply app/api/schema/users.py]: Okay. Will keep it that way. Will be using the account email here.

File: app/api/schema/users.py [hunk 2]
```diff
@@ -69,6 +69,17 @@ class Meta:
...
+    country = fields.Str(allow_none=True)
+    tax_info = fields.Str(allow_none=True)
+    company = fields.Str(allow_none=True)
+    company_address = fields.Str(allow_none=True)
+    company_city = fields.Str(allow_none=True)
+    zip_code = fields.Str(allow_none=True)
+    additional_info = fields.Str(allow_none=True)
+    company_comments = fields.Str(allow_none=True)
```
Comment (CosmicCoder96) [inline app/api/schema/users.py]: Rename this to additional_billing_info

File: app/api/schema/users.py [hunk 3]
```diff
@@ -69,6 +69,17 @@ class Meta:
     deleted_at = fields.DateTime(dump_only=True)
     details = fields.Str(allow_none=True)
     contact = fields.Str(allow_none=True)
+    contact_name = fields.Str(allow_none=True)
+    phone = fields.Str(allow_none=True)
+    email = fields.Str(allow_none=True)
+    country = fields.Str(allow_none=True)
+    tax_info = fields.Str(allow_none=True)
+    company = fields.Str(allow_none=True)
+    company_address = fields.Str(allow_none=True)
```
Comment (CosmicCoder96) [inline app/api/schema/users.py]: rename this to billing_address

File: app/api/schema/users.py [hunk 4]
```diff
@@ -69,6 +69,17 @@ class Meta:
...
+    company_city = fields.Str(allow_none=True)
```
Comment (CosmicCoder96) [inline app/api/schema/users.py]: rename this to `billing_city`

File: app/api/schema/users.py [hunk 5]
```diff
@@ -69,6 +69,17 @@ class Meta:
...
+    zip_code = fields.Str(allow_none=True)
```
Comment (CosmicCoder96) [inline app/api/schema/users.py]: billing_zip_code

File: app/api/schema/users.py [hunk 6]
```diff
@@ -69,6 +69,16 @@ class Meta:
     deleted_at = fields.DateTime(dump_only=True)
     details = fields.Str(allow_none=True)
     contact = fields.Str(allow_none=True)
+    contact_name = fields.Str(allow_none=True)
+    phone = fields.Str(allow_none=True)
```
Comment (CosmicCoder96) [inline app/api/schema/users.py]: billing_phone

File: app/api/schema/users.py [hunk 7]
```diff
@@ -69,6 +69,16 @@ class Meta:
...
+    country = fields.Str(allow_none=True)
```
Comment (CosmicCoder96) [inline app/api/schema/users.py]: billing_country

**Comment by mrsaicharan1** [comment]
> There is no migration file?

Was going to add it once https://github.com/fossasia/open-event-server/pull/6098 has been merged. But yeah, can add it now too.

**Comment by CosmicCoder96** [comment]
@mrsaicharan1 Yeah wait for it, then you will, either way, have to refactor.

File: migrations/versions/4279c2ac565b_.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '4279c2ac565b'
+down_revision = 'b08a4ffff5dd'
```
Comment (iamareebjamal) [inline migrations/versions/4279c2ac565b_.py]: Fix these to be latest

**Comment by mrsaicharan1** [comment]
@iamareebjamal Updated to the latest revision. @shreyanshdwivedi @uds5501

**Pull request merged**